src/heroku/orb.yml

```diff
@@ -111,6 +121,10 @@ commands:
             if [[ "<< parameters.only-branch >>" == "" ]] || [[ "${CIRCLE_BRANCH}" == "<< parameters.only-branch >>" ]]; then
               git push $force https://heroku:<< parameters.api-key >>@git.heroku.com/<< parameters.app-name >>.git << parameters.branch >>:master
             fi
+      - when:
+          condition: << parameters.maintenance-mode >>
+          steps:
+            - run: heroku maintenance:off --app << parameters.app-name >>
```

Comment:
In my configs (before this PR made the maintenance option available in commands), I had this clause to ensure we exit maintenance mode if a deploy fails:

```
-          when: always
```